### ISSUE TITLE
Alternating Flux Crashes on Start-up

### ISSUE THREAD

**Issue by coldReactive** [AntiBlueQuirk/alternatingflux#11]
The game crashed whilst there was a severe problem during mod loading that has caused the game to fail
The game crashed whilst there was a severe problem during mod loading that has caused the game to fail
Error: net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from Alternating Flux (alternatingflux)

Crash Log: https://pastebin.com/kyN1axEN

**Comment by ClaudiusMinimus** [comment]
Crashed only after I updated to Immersive Engineering 0.12 - 89

**Comment by brisingraerowing** [comment]
Duplicate of #8. The fix has not been released yet.

**Comment by raindropworks** [comment]
Appears the fix was merged in #9 but just hasn't been packaged and pushed to Curseforge?

**Comment by coldReactive** [comment]
Yeah, the curse version hasn't been updated since September 1st 2018.

**Comment by JadianRadiator** [comment]
Can't wait for the fixed version to get pushed out to Curse.

**Comment by elijahlorden** [comment]
I love this mod, hopefully the fixed version will be pushed soon!

**Comment by mindcloud69** [comment]
Is this project even being maintained any more?

**Comment by elijahlorden** [comment]
Is it possible for somebody else to build the mod?

**Comment by politespectre** [comment]
Does anyone know how to fix the curseforge version after downloading or download the fixed version?

**Comment by raindropworks** [comment]
Seems @AntiBlueQuirk hasn't posted anything on GitHub in a couple of weeks, but they've been active this month.  Not sure why a fix that was merged a month ago hasn't made it to Curseforge yet, and unfortunately, I don't know how to properly package for doing it ourselves

**Comment by TheDeviantCrafter** [comment]
Alternating Flux updated a week ago, with this changelog:

`Fixed bug with IE 0.12-89 - Fix authored by BrisingrAerowing`

Is this issue fixed?

**Comment by brisingraerowing** [comment]
I am using the newest version, and it works for me.
